Containerfile

```diff
@@ -23,6 +22,6 @@ FROM ghcr.io/gardenlinux/gardenlinux:1592
 EXPOSE 8080
 
 COPY --from=build /tinysapmachine /jre
-COPY --from=build /usr/src/glvd/glvd-0.0.1-SNAPSHOT.jar /
+COPY --from=build /usr/src/glvd/glvd-0.0.1-SNAPSHOT /
 
-CMD ["/jre/bin/java", "-jar", "/glvd-0.0.1-SNAPSHOT.jar"]
+CMD ["/jre/bin/java", "-jar", "/glvd-0.0.1-SNAPSHOT/glvd-0.0.1-SNAPSHOT.jar"]
```

Comment:
maybe we could define the version some place central in this repo?

Reply:
Yes.

I've intentionally pushed versioning for glvd down the road, from my perspective it does not make sense (yet) to care about versioning, and in a cloud product it is anyway more complex because the api needs to have the right version of the database container, the right version of the db schema, etc.

You're right with that comment. To be open I think it's not urgent, that's why I'd keep it like that for now.